pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -206,7 +206,16 @@ func (sv *openShiftClusterStaticValidator) validateMasterProfile(path string, mp
 	switch mp.VMSize {
 	case VMSizeStandardD8sV3,
 		VMSizeStandardD16sV3,
-		VMSizeStandardD32sV3:
+		VMSizeStandardD32sV3,
+		VMSizeStandardD8asV4,
```

Comment:
I think we selected a limited set of master VM sizes to ensure they'd have sufficient IOPS to run etcd. In that case, I'm going to assume we need a minimum of 16k IOPS, from the three supported node sizes here: https://docs.microsoft.com/en-us/azure/openshift/support-policies-v4#master-nodes

I think the rest are okay, but `Standard_D8a_v4` can only manage 12k IOPS, and should probably be removed: https://docs.microsoft.com/en-us/azure/virtual-machines/dav4-dasv4-series#dav4-series

Reply:
ok, but this brings a challenge as there is number of regions where these VMS are not available.